lib/ansible/modules/system/user.py

```diff
@@ -2859,9 +2859,9 @@ def main():
             ssh_key_file=dict(type='path'),
             ssh_key_comment=dict(type='str', default=ssh_defaults['comment']),
             ssh_key_passphrase=dict(type='str', no_log=True),
-            update_password=dict(type='str', default='always', choices=['always', 'on_create']),
+            update_password=dict(type='str', default='always', choices=['always', 'on_create'], no_log=False),
```

Comment:
These changes to the `user` module are good, but should be done in a separate PR.

Reply:
Agreed, but just to confirm, the only reason I made this change was that, as the current tests are written for the `user` module, the existence of any warning fails the tests, which fails CI. Is this okay?

Reply:
We'll need to merge the user module PR before this one. Once that PR is merged, tests for this PR will pass.

Reply:
Pull request #66219 has been created. Is it okay to mark this as resolved?

Reply:
> Pull request #66219 has been created. Is it okay to mark this as resolved?

Yup.

> Also, please let me know if anything needs to be done or redone.

You can remove the changes to the `user` module from this PR.

> I'm not the most familiar with GitHub's tools yet and I'm already regretting not creating a branch for this pull request.

Creating new branches in your fork for a PR isn't required but it does make it easier to segment work.